### PR TITLE
CADC-10067 - add Storage Inventory and decompression support, replace preview and thumbnail generation logic.

### PR DESCRIPTION
**Pull request by SharonGoliath**

### CODE REVIEW

File: omm2caom2/tests/data/C190618_0336_SCI.xml [hunk 1]
```diff
@@ -49,98 +49,9 @@
         <caom2:runID>190618</caom2:runID>
         <caom2:reference>http://genesis.astro.umontreal.ca</caom2:reference>
       </caom2:provenance>
-      <caom2:position>
```
Comment (andamian) [inline omm2caom2/tests/data/C190618_0336_SCI.xml:52]: Why was this removed? Not relevant for the test?

File: omm2caom2/tests/test_omm_name.py [hunk 2]
```diff
@@ -66,79 +66,124 @@
...
-        fname_on_disk='C121212_00001_SCI.fits',
-        file_name='C121212_00001_SCI.fits.gz',
-    )
-    assert test_subject.is_valid()
-    assert test_subject.obs_id == 'C121212_00001'
-    assert (
-        test_subject.file_uri == 'ad:OMM/C121212_00001_SCI.fits.gz'
-    ), 'wrong file uri'
+        with pytest.raises(CadcException):
+            test_subject = OmmName._add_extensions('C121212_00001_SCI')
```
Comment (andamian) [inline omm2caom2/tests/test_omm_name.py]: I'm not sure whether this was the intention, but this succeeds when the first of the calls raises the exception.

**Review by andamian — APPROVED** [review]

File: omm2caom2/tests/data/C190618_0336_SCI.xml (hunk 1 above)
Comment (SharonGoliath) [reply omm2caom2/tests/data/C190618_0336_SCI.xml:52]: That's correct. It's metadata that's generated server-side and irrelevant to the test comparison. I was making the test data smaller.

File: omm2caom2/tests/test_omm_name.py (hunk 2 above)
Comment (SharonGoliath) [reply omm2caom2/tests/test_omm_name.py]: Thanks for catching this - I've split the cases out into separate tests now.
